Models/CompanyProperties.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DIS_projekt.Models
+{
+    public class CompanyProperties
+    {
+        public int Id { get; set; }
+        public string CompanyName { get; set; }
+        public string EmployeeName { get; set; }
```

Comment:
Ako opisuješ firmu, tvoja firma može imati samo jednog zaposlenika. Možda da promijeniš ovo u ime vlasnika ili nešto drugo